packages/bezier-react/src/hooks/useToken.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import { tokens } from '@channel.io/bezier-tokens'
+
+import { createContext } from '~/src/utils/reactUtils'
+
+type Tokens = typeof tokens
+type GlobalTokens = Tokens['global']
+type SemanticTokens = Omit<Tokens, 'global'>
+
+export type TokenContextValue = GlobalTokens & SemanticTokens[keyof SemanticTokens]
+
+const [TokenContextProvider, useTokenContext] = createContext<TokenContextValue | null>(null, 'TokenProvider')
+
+// TODO: Change theme name constant to import from bezier-tokens
+export type ThemeName = 'light' | 'dark'
+
+const tokenSet: Record<ThemeName, TokenContextValue> = Object.freeze({
+  light: {
+    ...tokens.global,
+    ...tokens.lightTheme,
+  },
+  dark: {
+    ...tokens.global,
+    ...tokens.darkTheme,
+  },
+})
```

Comment:
satisfies 를 활용하면 context 로 넘기는 tokenSet의 타입을 좁힐 수 있을 것 같습니다

```suggestion
const tokenSet = Object.freeze({
  light: {
    ...tokens.global,
    ...tokens.lightTheme,
  },
  dark: {
    ...tokens.global,
    ...tokens.darkTheme,
  },
}) satisfies Record<ThemeName, FlattedTokens>
```

Reply:
아 동치군요 착각했습니다